bundles/GoogleMarketingBundle/public/js/report/analytics/siteoverview.js

```diff
@@ -167,7 +167,7 @@ pimcore.report.analytics.overview = Class.create(pimcore.report.abstract, {
 });
 
 // add to report broker
-if(pimcore.bundle && pimcore.bundle.customreports) {
-    pimcore.bundle.customreports.broker.addGroup("analytics", "google_analytics", "pimcore_icon_analytics");
-    pimcore.bundle.customreports.broker.addReport(pimcore.report.analytics.overview, "analytics");
-}
+
+pimcore.bundle.googlemarketing.report.broker.addGroup("analytics", "google_analytics", "pimcore_icon_analytics");
```

Comment:
having its own broker will not work anymore with custom reports...right? https://github.com/pimcore/pimcore/blob/6b7b4ee668fa7c5907c26d068eee20ca28f4388d/bundles/CustomReportsBundle/public/js/pimcore/report/panel.js#L85

how this is used then?

Reply:
Changed here https://github.com/pimcore/pimcore/pull/14276/commits/a63ab491712bad94ed9159aad3f6b15594c25399 to use CustomReportsBundle broker